client/src/components/Messaging/Message.js

```diff
@@ -42,6 +42,11 @@ export const Message = ({
         <div className={`${styles.messageContainer} ${!darkMode && styles.lightModeContainer}`}>
           {image && <Image src={image} maxWidth="300px" maxHeight="300px" />}
           {body}
+          {id !== undefined ? (
+            <span className={styles.timestamp}>{new Date(id).toLocaleString()}</span>
```

Comment:
Better to use ternary operator something like:
`timestamp && <span className={styles.timestamp}>{new Date(id).toLocaleString()}</span>`

Reply:
@abhishek-aa Also as it's mostly for short lived chat, lets just show only hour and min i.e. 22:10

Reply:
I'm on it!